### ISSUE TITLE
Can't add nic to a vm

### ISSUE THREAD

**Issue by john-kheir** [zero-os/0-core#466]
```
client.kvm.add_nic(vm_uuid, 'vlan', id=str(t1))
```
## When trying to add nic to vm, i get this error

```
----------------------------------------------------------------------

Traceback (most recent call last):

  File "/home/travis/build/zero-os/0-core/tests/testsuite/b_extended/tests_02_machines.py", line 66, in test001_kvm_add_remove_nics

    self.client.kvm.add_nic(vm_uuid, 'vlan', id=str(t1))

  File "/home/travis/virtualenv/python3.5.3/lib/python3.5/site-packages/zeroos/core0/client/client.py", line 2237, in add_nic

    return self._client.json('kvm.add_nic', args)

  File "/home/travis/virtualenv/python3.5.3/lib/python3.5/site-packages/zeroos/core0/client/client.py", line 726, in json

    result = self.sync(command, arguments, tags=tags, id=id)

  File "/home/travis/virtualenv/python3.5.3/lib/python3.5/site-packages/zeroos/core0/client/client.py", line 716, in sync

    raise RuntimeError('invalid response: %s' % result.state, result)

RuntimeError: ('invalid response: ERROR', STATE: ERROR

STDOUT:

STDERR:

DATA:

"failed to attach device: virError(Code=1, Domain=0, Message='internal error: Unable to add port tap0 to OVS bridge 21e4e1b47')"

)

-------------------- >> begin captured logging << --------------------
```

## Note
OVS container seems to be corrupted 
```
In [35]: ovs.system('ovs-vsctl show').get()
Out[35]: 
STATE: ERROR
STDOUT:

STDERR:
ovs-vsctl: ../lib/poll-loop.c:111: assertion !fd != !wevent failed in poll_create_node()

DATA:


```
## Version
master @Revision: 05614269b8c46e24e1284cf3d7210c0f243fe4e5

**Comment by muhamadazmy** [comment]
Fixed by #468

**Comment by john-kheir** [comment]
fixed
